releases/v1.2.5.toml

```diff
@@ -0,0 +1,38 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.2.4"
+
+pre_release = false
+
+preface = """\
+The fifth patch release for `containerd` 1.2 contains fixes
+for image management, cgroup management and the
+CRI plugin.
+It also updates runc to include an improved fix for CVE-2019-5736
+to reduce the increased memory-consumption introduced by the original
+patch.
+All these changes are noted below.
+### Notable Updates
+* Fix an issue that non-existant parent directory in image layers is created with permission
+`0700`. https://github.com/containerd/containerd/issues/3017
+* Fix an issue that snapshots of the base image can be deleted by mistake, when images
+built on top of it are deleted. https://github.com/containerd/containerd/pull/3087
+* cri: Fix a bug that pod can't get started when the same volume is defined
+differently in the image and the pod spec. https://github.com/containerd/cri/issues/1059
+* cgroups updated to dbea6f2bd41658b84b00417ceefa416b97 to fix issues for systemd 420 and
+non-existant cgroups. https://github.com/containerd/containerd/pull/3079
+* runc updated to 2b18fe1d885ee5083ef9f0838fee39b62d653e30 to include the improved
+fix for CVE-2019-5736. https://github.com/containerd/containerd/pull/3082
+* cri updated to 8e7ca12f411d65de58ca672e8e4a0c1464b4fe34. https://github.com/containerd/containerd/pull/3084
+"""
```

Comment:
`* support for GC references to content from snapshot and container objects. https://github.com/containerd/containerd/pull/3080`

Reply:
Done